ttnn/cpp/ttnn/operations/experimental/plusone/plusone_pybind.cpp

```diff
@@ -0,0 +1,44 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "ttnn/cpp/pybind11/decorators.hpp"
+
+#include "ttnn/operations/experimental/plusone/plusone.hpp"
+#include "ttnn/operations/experimental/plusone/plusone_pybind.hpp"
+
+namespace ttnn::operations::experimental::plusone::detail {
+namespace py = pybind11;
+void bind_experimental_plusone_operation(py::module& module) {
+    auto doc =
+        R"doc(plus_one(input_tensor: ttnn.Tensor) -> ttnn.Tensor
+
+            Returns input tensor elementes increased by 1.
+            Input tensor must have UINT32 data type and ROW_MAJOR layout.
+
+            Equivalent pytorch code:
+
+            .. code-block:: python
+
+                return torch.add(input_tensor, 1)
+
+            Args:
+                * :attr:`input_tensor`: Input Tensor for plusone.
+
+        )doc";
```

Comment:
It would be nice to include some sort comment about being preferable for small tensors.